hardhat.config.ts

```diff
@@ -17,8 +17,30 @@ if (walletSecret === 'undefined') {
 }
 const accounts = walletSecret.length === 64 ? [walletSecret] : { mnemonic: walletSecret };
 
-const mainnetEtherscanKey = process.env.MAINNET_ETHERSCAN_KEY;
-const testnetEtherscanKey = process.env.TESTNET_ETHERSCAN_KEY;
+let testnetEtherscanKey;
+let mainnetEtherscanKey;
+
+if (
```

Comment:
This can be simplified as:
```js
testnetEtherscanKey = process.env.TESTNET_ETHERSCAN_KEY;
if (
  testnetEtherscanKey === null ||
  testnetEtherscanKey < 34
) {
  console.log('API Key does not exist for testnet');
}
```

The same for the mainnet.